### PR TITLE
Update export-pkg error description

### PR DESCRIPTION
**Pull request by uilianries**
Related to PR https://github.com/conan-io/conan/pull/6720

### CODE REVIEW

File: reference/commands/creator/export-pkg.rst [hunk 1]
```diff
@@ -86,6 +86,8 @@ different sources for the files:
    and the ``self.copy(...)`` functions will copy matching files from the ``source_folder``
    **and** ``build_folder`` to the corresponding path in the Conan cache (working directory
    corresponds to the ``build_folder``).
+ * If the arguments ``--package-folder``, ```--build-folder`` or ``--source-folder`` are
+   declared, but the path is incorrect, :command:`export-pkg` will raise a ``ConanException``.
```
Comment (danimtb) [inline reference/commands/creator/export-pkg.rst]: I don't think the exception type is important here
```suggestion
   declared, but the path is incorrect, :command:`export-pkg` will raise an exception.
```

**Comment by danimtb** [comment]
Travis is green but here it is yellow. What happened there?

**Comment by uilianries** [comment]
Restarting ...

**Comment by danimtb** [comment]
same...

**Comment by uilianries** [comment]
something is wrong, even restarting is not working. I could close this PR and open a new one.

**Comment by danimtb** [comment]
@uilianries try pushing an empty commit here

**Comment by uilianries** [comment]
@danimtb it worked!

**Pull request merged**